### PR TITLE
add recipe for datajoint

### PR DESCRIPTION
**Pull request by tjd2002**
Fixes: https://github.com/datajoint/datajoint-python/issues/293
Builds locally on OSX.

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages
- [X] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tjd2002** [comment]
@dimitri-yatsenko, can you make a comment here indicating whether you agree to be a maintainer of this feedstock? (This will also allow you to add other maintainers, e.g. other DataJoint devs)

**Comment by guzman-raphael** [comment]
@tjd2002 great job! Actually @dimitri-yatsenko's preference is for me to be the maintainer. You agree to be the maintainer.

**Comment by tjd2002** [comment]
@guzman-raphael Great! I'll leave myself listed as a maintainer for now, so I can help with the initial packaging, but I'll assume that you will take on the actual maintainer role (updating for new releases, etc).

@conda-forge/help-python : please review this PR, and thank you!

File: recipes/datajoint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  sha256: a35e8d87398b813aef95f4a1bfbaacae5cc905971d6cb99d3114ab2c0f35229b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.4
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml]: There needs to be a space here else feedstock conversion will fail
```suggestion
    - python >=3.4
```
(there should not be a space between `>=` and `3.4`)

File: recipes/datajoint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - pip
+  run:
+    - python>=3.4
+    - minio
+    - pyparsing
+    - networkx
+    - pymysql>=0.7.2
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml]:
```suggestion
    - pymysql >=0.7.2
```

File: recipes/datajoint/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.4
+    - pip
+  run:
+    - python>=3.4
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml]:
```suggestion
    - python >=3.4
```

File: recipes/datajoint/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    - datajoint
+
+about:
+  home: https://datajoint.io
+  license: LGPL-2.1
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: 'A relational data framework for scientific data pipelines with MySQL backend.'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml]: This comment should be removed
```suggestion
```

File: recipes/datajoint/meta.yaml~ [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml~]: This file should be removed. It is probably a backup file from your text editor.

File: recipes/datajoint/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a35e8d87398b813aef95f4a1bfbaacae5cc905971d6cb99d3114ab2c0f35229b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (chrisburr) [inline recipes/datajoint/meta.yaml]: It doesn't really matter but `--no-deps` is no longer required:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment (tjd2002) [reply recipes/datajoint/meta.yaml]: OK. The docs still suggest it: https://conda-forge.org/docs/maintainer/adding_pkgs.html, and I've been burned before by leaving it out (https://github.com/conda-forge/conda-forge.github.io/issues/684), so I think I'll leave it in if the recipe is still acceptable.

**Pull request merged**